File: src/Services/Announcement/Announcement.Common/Models/Announcement.cs
using System;
using System.Collections.Generic;

namespace Announcement.Common.Models
{
    public class Announcement
    {
        public int Id { get; set; }
        public int EmployeeId { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public DateTime DateTime { get; set; }

        public Employee Employee { get; set; }
        public List<ClassAnnouncement> ClassAnnouncements { get; set; }
    }
}

File: src/Web/Spa/src/app/models/announcement.ts
import { ShortEmployee } from './employee';
import { Class } from './class';


export class Announcement{
    id:number;
    title:string;
    body:string;
    dateTime:Date;
    employee:ShortEmployee;
}

export class ClassAnnouncement{
    announcement:Announcement;
    class:Class;
}